src/odemis/driver/xt_client.py

```diff
@@ -1616,6 +1622,65 @@ def _get_acq_msg(self, **kwargs):
         return msg
 
 
+class Chamber(model.Actuator):
+    """
+    Component representing the vaccum chamber. Changing the pressure is possible by moving the "vacuum" axis,
+    which accepts two position values: 0 for vented, 1 for vacuum.
+    """
+
+    def __init__(self, name, role, parent, **kwargs):
+        axes = {"vacuum": model.Axis(choices={0: "vented", 1: "vacuum"})}
+        model.Actuator.__init__(self, name, role, parent=parent, axes=axes, **kwargs)
+
+        self.position = model.VigilantAttribute({"vacuum": 0}, readonly=True)
+        self.pressure = model.FloatContinuous(self.parent.get_pressure(), range=(0, 1e9), readonly=True, unit="Pa")
```

Comment:
1 billion Pa sounds a little over the top for a vacuum chamber. The xtlib does have a function to report the pressure range. I think the right way would be to extend the xtlib-adapter to provide a `pressure_info()` with this range (and unit).